app/components/application_record_card/view.rb

```diff
@@ -8,13 +8,12 @@ class View < GovukComponent::Base
 
     with_collection_parameter :record
 
-    attr_reader :record, :heading_level, :show_provider, :hide_progress_tag
+    attr_reader :record, :heading_level, :current_user
 
-    def initialize(heading_level = 3, record:, show_provider: false, hide_progress_tag: false)
+    def initialize(heading_level = 3, record:, current_user:)
```

Comment:
Passing the whole `current_user` into the component kinda goes against our "dumb components" mantra - I might prefer for the component to receive two separate configurable options `show_provider` and `hide_progress_tag` 🤔  up for discussion though!

Reply:
@mooktakim, is this something we can explore/have explored in your PR?

Reply:
@AbigailMcP 
The logic for those old parameters is getting really complicated. It's best to do that logic inside the component class, instead of the view.